app/about_us/OurPartners.js

```diff
@@ -0,0 +1,51 @@
+import styles from "./partners.module.css";
+
+const OurPartners = () => {
+
+const CompanyName = [
```

Comment:
```suggestion
const companyName = [
```
Capitalized variable names are reserved for components.